### ISSUE TITLE
[Accessibility][Screen Readers-Filter Dialog] : Focus should move to the chart section after selecting any dialog.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2056]
**User Experience** 
Users will find it difficult to navigate through the application if focus order is incorrect. Here, focus should move to the flow chart section after selecting any of the dialog otherwise user would have to navigate through all the dialogs which will be time consuming.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and set the zoom at 400%.
2. Select 'Design Flow' from left navigation.
3. Now, select any dialog available in the filter dialog pane.
4. Observe the issue.

**Actual:** 
When dialog gets selected, and user presses tab key, focus moves to the next dialog available in list item instead of the flow chart section.

**Expected:**
On selecting any of the dialog, the focus should move to the flow chart of the dialog so that the user knew about the opened chart.

**Note:**
1. Same issue found throughout the page, on activating any of the node property pane gets opened but the focus remains on the chart and when user press next tab new property pane gets opened. Due to this the keyboard user will not be able to reach property pane. I suggest the role should be assigned to the node or any shortcut should be defined to reach directly to the property pane.
[MAS2.4.3_Property pane.zip](https://github.com/microsoft/BotFramework-Composer/files/4241368/MAS2.4.3_Property.pane.zip)
 

**MAS Impacted:** [MAS2.4.3](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9})

**Attachment:**
[MAS2.4.3_Focus should move to the first interactive element.zip](https://github.com/microsoft/BotFramework-Composer/files/4241053/MAS2.4.3_Focus.should.move.to.the.first.interactive.element.zip)

**Comment by ashish315** [comment]
@cwhitten As checked the issue is not fixed yet as well as now the focus is not going inside the chart section with Tab key. On pressing tab from 'Show Code' the focus should come inside the tree pane but the focus stuck on the whole pane. As well as when we use mouse to take focus inside the pane the keyboard focus get tapped inside the chart section.
Please refer the video's for the same "[Issue.zip](https://github.com/microsoft/BotFramework-Composer/files/4524669/Issue.zip)"

I downloaded below Master build file last updated on 23rd April.

**Test Environment**
Browser: Microsoft Edge Dev {Version 84.0.488.1 (Official build) dev (64-bit)}
OS build: 2004 (19562.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
![Github](https://user-images.githubusercontent.com/60913131/80138432-6f8eca80-85c2-11ea-88d4-e617ae22ae4f.png)

**Comment by beyackle** [comment]
I think what's going on here is a confusion of two different issues. The original one was about the focus moving into the visual editor after selecting a dialog, and that has in fact been fixed.

However, there's a separate problem where Tab and Shift-Tab no longer work inside the visual editor because those keys have been forcibly remapped in ways that break at least Narrator - the visual-editor extension has a whole `KeyboardZone` component that overwrites the usual Tab button behavior with its own.

**Comment by ashish315** [comment]
@beyackle The original issue is not fixed yet! Please check and validate.

**Comment by beyackle** [comment]
I think there's some confusion here over the word "select". I had been interpreting it to mean "move to and hit Enter", selecting a dialog for display in the main window, and that case does move focus over. However, moving through the list of dialogs (without hitting Enter, just going back and forth through the list) does in fact use Tab instead of the arrow keys, and you are correct that this is unexpected behavior, given how things like the sidebar nav menu use the arrow keys.

I'll go ahead and fix that, because that is a problem; is that also the behavior you were referring to?

**Comment by ashish315** [comment]
Yup! Sounds cool. The user should be able to navigate with arrow key and once hit enter the focus should jump to the opened chart.

**Comment by Amit8527510735** [comment]
#Nonbenchmark;
